core/src/main/java/org/frankframework/filesystem/SftpFileSystem.java

```diff
@@ -58,6 +63,7 @@ public class SftpFileSystem extends SftpSession implements IWritableFileSystem<S
 	@Override
 	public void open() throws FileSystemException {
 		ftpClient = openClient(remoteDirectory);
+		lastCheck = System.currentTimeMillis();
```

Comment:
For this purpose it might be better to get the nano-time as that is monotonically increasing, guaranteed regardless of time-corrections or daylight-savings time.

You only care about intervals and not about wall-clock time, right?

Reply:
Interesting thoughts. I wasn't aware of this, while adding it. 
I just did some investigation in this topic of the tradeoffs. I've read this article: https://www.geeksforgeeks.org/java-system-nanotime-vs-system-currenttimemillis/
I think for this solution, it is better to keep the `currentTimeMillis`, since it is faster and high precision is not required. Con: I might will skip checking when the wall clock goes back from 3 to 2 am, while daylight-saving corrections are made. In all other situations, it will perhaps do an extra check, but that's fine. It will benefit the rest of the year of the speed of the check. 
Or do you have any second thoughts?

Reply:
As discussed in the meeting, we keep it like it is now.